src/pages/multipleProductValidity.tsx

```diff
@@ -40,113 +39,148 @@ const MultipleProductValidity = ({
-                                        <td className="govuk-table__cell">£{product.productPrice}</td>
-                                        <td className="govuk-table__cell">
-                                            {`${product.productDuration} ${product.productDurationUnits}${
-                                                Number(product.productDuration) > 1 ? 's' : ''
-                                            }`}
-                                        </td>
-                                        <td className="govuk-table__cell">
-                                            <span className="govuk-radios validity-select-wrapper">
-                                                <span className="govuk-radios__item">
+                                {multipleProducts.map((product, index) => {
+                                    return (
+                                        <tr className="govuk-table__row" key={product.productNameId}>
+                                            <td className="govuk-table__cell">{product.productName}</td>
+                                            <td className="govuk-table__cell">£{product.productPrice}</td>
+                                            <td className="govuk-table__cell">
+                                                {`${product.productDuration} ${product.productDurationUnits}${
+                                                    Number(product.productDuration) > 1 ? 's' : ''
+                                                }`}
+                                            </td>
+                                            <td className="govuk-table__cell">
+                                                <FormElementWrapper
+                                                    errors={errors}
+                                                    errorId={`validity-option-${index}`}
+                                                    errorClass="govuk-select--error"
+                                                    hideText
+                                                >
+                                                    <>
+                                                        {/* eslint-disable-next-line jsx-a11y/no-onchange */}
+                                                        <select
+                                                            className="govuk-select farestage-select"
+                                                            id={`validity-option-${index}`}
+                                                            name={`validity-option-${index}`}
+                                                            aria-labelledby={`stop-name-header stop-${index} naptan-code-header naptan-${index}`}
+                                                            onChange={handleSelection}
+                                                            defaultValue={product.productValidity}
```

Comment:
do we need to use value instead of defaultValue here? @dannydaviesIW mentioned problems with the MatchingBase around this.